csharp-basics/exercises/Arrays/Arrays/TicTacToe/Program.cs

```diff
@@ -6,25 +6,117 @@ class Program
+                       || (board[2, 0] == oPlayer && board[2, 1] == oPlayer && board[2, 2] == oPlayer)
+                       || (board[2, 2] == oPlayer && board[1, 1] == oPlayer && board[0, 0] == oPlayer)
+                       || (board[0, 2] == oPlayer && board[1, 1] == oPlayer && board[2, 0] == oPlayer)
+                       || (board[0, 0] == oPlayer && board[1, 0] == oPlayer && board[2, 0] == oPlayer)
+                       || (board[0, 1] == oPlayer && board[1, 1] == oPlayer && board[2, 1] == oPlayer)
+                       || (board[0, 2] == oPlayer && board[1, 2] == oPlayer && board[2, 2] == oPlayer))
+                    {
+                        Console.WriteLine("O won!");
+                        break;
+                    }
+                    else if (tie == 9)
+                    {                       
+                        Console.WriteLine("It's a tie");
+                        break;
+                    }
+                   
+                    Console.Write("Please choose line where you want to put 0: ");
+                    var userInputOLine = int.Parse(Console.ReadLine());
+                    Console.Write("Please choose column where you want to put 0: ");
+                    var userInputOColumn = int.Parse(Console.ReadLine());
+                    board[userInputOLine, userInputOColumn] = oPlayer;
+                    DisplayBoard();
+                    tie++;
+
+                    if((board[0, 0] == xPlayer && board[0, 1] == xPlayer && board[0, 2] == xPlayer)
+                       || (board[1, 0] == xPlayer && board[1, 1] == xPlayer && board[1, 2] == xPlayer)
+                       || (board[2, 0] == xPlayer && board[2, 1] == xPlayer && board[2, 2] == xPlayer)
+                       || (board[2, 2] == xPlayer && board[1, 1] == xPlayer && board[0, 0] == xPlayer)
+                       || (board[0, 2] == xPlayer && board[1, 1] == xPlayer && board[2, 0] == xPlayer)
+                       || (board[0, 0] == xPlayer && board[1, 0] == xPlayer && board[2, 0] == xPlayer)
+                       || (board[0, 1] == xPlayer && board[1, 1] == xPlayer && board[2, 1] == xPlayer)
+                       || (board[0, 2] == xPlayer && board[1, 2] == xPlayer && board[2, 2] == xPlayer))
+                    {
+                        Console.WriteLine("X won!");
+                        break;
+                    }
+                    else if((board[0, 0] == oPlayer && board[0, 1] == oPlayer && board[0, 2] == oPlayer)
+                       || (board[1, 0] == oPlayer && board[1, 1] == oPlayer && board[1, 2] == oPlayer)
+                       || (board[2, 0] == oPlayer && board[2, 1] == oPlayer && board[2, 2] == oPlayer)
+                       || (board[2, 2] == oPlayer && board[1, 1] == oPlayer && board[0, 0] == oPlayer)
+                       || (board[0, 2] == oPlayer && board[1, 1] == oPlayer && board[2, 0] == oPlayer)
```

Comment:
Tev šis atkārtojas

Reply:
Neredzu labojumus. Var izveidot metdoi, kur padod x vai o un tad pārbauda vai ir uzvarētājs.. un uzvrs tas, kurš tika padots metodē